tests/test_sqlalchemy.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import sqlalchemy.exc
+from sqlalchemy import MetaData
+from sqlalchemy.engine import Engine
+from sqlalchemy.sql.expression import TextClause
+
+
+def test_create_view(snowflake_engine: Engine):
```

Comment:
mmm. I'll add an additional test for what you're asking for, but I'm personally specifically interested in knowing that SQLAlchemy succeeds here (which obv we know right now calls `.description`, but still

Reply:
sqlalchemy's behaviour is also covered [here](https://github.com/tekumara/fakesnow/blob/54e661fc8510d8d0c2be60c90f3d7235a5e2a20b/tests/test_fakes.py#L987). But if you think there's something not covered, or its worth having a basic "hello world" type integration test that does a basic create, insert, select from a table using sqlalchemy, that would be fine.

Reply:
FYI sqlalchemy's behaviour is also covered [here](https://github.com/tekumara/fakesnow/blob/54e661fc8510d8d0c2be60c90f3d7235a5e2a20b/tests/test_fakes.py#L987). But if you think there's something not covered, or its worth having a basic "hello world" type integration test that does a basic create, insert, select from a table using sqlalchemy, that would be good as it would extend coverage.

Reply:
FYI sqlalchemy's behaviour is also covered [here](https://github.com/tekumara/fakesnow/blob/54e661fc8510d8d0c2be60c90f3d7235a5e2a20b/tests/test_fakes.py#L987). But if you think there's something not covered, or its worth having a basic "hello world" type integration test, like a basic create, insert, select from a table using sqlalchemy, that would be good as it would extend coverage.